modules/auxiliary/scanner/kerberos/kerberos_login.rb

```diff
@@ -0,0 +1,44 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
```

Comment:
This new `kerberos_login` module is based on the naming convention used for all of our other protocols.

For now kerberos_login/kerberos_enumusers call off to the same mixin, and I made an intentional decision to leave the old module kerberos_enumusers around so it's still searchable by users. Metasploit has a way to alias modules, but it doesn't wire up in a great way to the search functionality of msfconsole - it might be confusing to users that just want to enumerate accounts/follow old training material.

For now this seems like the simplest solution in terms of discoverability/clarity to end users, and will allow for the module implementations to diverge again in the future if need be.